### PR TITLE
Add time stamp in 'time ago' format to Activity items

### PR DESCRIPTION
**Pull request by Patrick1904**
https://github.com/nearprotocol/near-wallet/issues/242

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "connected-react-router": "^6.6.0",
     "fetch-send-json": "0.0.2",
     "js-sha256": "^0.9.0",
+    "moment": "^2.24.0",
```
Comment (vgrichina) [inline package.json]: https://bundlephobia.com/result?p=moment@2.24.0 feels a bit heavy if we only going to use for time stamps.  Maybe worth checking alternatives?

**Comment by vgrichina** [comment]
https://bundlephobia.com/result?p=javascript-time-ago@2.0.4 seems like good alternative.

File: package.json (hunk 1 above)
Comment (Patrick1904) [reply package.json]: Changed to use: [](https://bundlephobia.com/result?p=timeago.js@4.0.1)

Comment (Patrick1904) [reply package.json]: Changed to use: https://bundlephobia.com/result?p=timeago.js@4.0.1

**Pull request merged**